client-app/shared/cart/components/add-to-cart.vue

```diff
@@ -90,6 +90,7 @@ const maxQuantity = computed(() => props.product.maxQuantity);
 const { cart, addToCart, changeItemQuantity } = useCart();
 const { t } = useI18n();
 const ga = useGoogleAnalytics();
+const { getTranslation } = useErrorsTranslator("validation_error.");
```

Comment:
Use only locale key prefix `validation_error` without dot. Dot symbol should be a part of composable logic.

Reply:
What if we have several dots? "shared.account.sign_in_form.errors."